api/tests/routes/backoffice_v3/custom_reimbursement_rule_test.py

```diff
@@ -151,3 +154,168 @@ def test_list_rules_more_than_max(self, authenticated_client):
         assert response.status_code == 200
         assert html_parser.count_table_rows(response.data) == 25  # extra data in second row for each booking
         assert "Il y a plus de 25 résultats dans la base de données" in html_parser.extract_alert(response.data)
+
+
+class CreateCustomReimbursementRulesTest(PostEndpointHelper):
+    endpoint = "backoffice_v3_web.reimbursement_rules.create_custom_reimbursement_rules"
+    needed_permission = perm_models.Permissions.CREATE_REIMBURSEMENT_RULES
+    tomorrow = datetime.date.today() + datetime.timedelta(days=1)
+    next_year = datetime.date.today() + datetime.timedelta(days=365)
+
+    def test_basics(self, authenticated_client):
+        offerer = offerers_factories.OffererFactory()
+        response = self.post_to_endpoint(
+            authenticated_client,
+            form={
+                "offerer": offerer.id,
+                "subcategories": ["ABO_CONCERT", "ABO_JEU_VIDEO"],
+                "rate": "12,34",
+                "start_date": self.tomorrow,
+                "end_date": self.next_year,
+            },
+        )
+        assert response.status_code == 303
+        rules = finance_models.CustomReimbursementRule.query.all()
+        assert len(rules) == 1
+        rule = rules[0]
+        assert rule.offererId == offerer.id
+        assert rule.rate == Decimal("0.1234")
+        assert rule.timespan.lower == datetime.datetime.combine(self.tomorrow, datetime.datetime.min.time())
+        assert rule.timespan.upper == datetime.datetime.combine(self.next_year, datetime.datetime.min.time())
+        assert rule.subcategories == ["ABO_CONCERT", "ABO_JEU_VIDEO"]
+
+    def test_invalid_offerer(self, authenticated_client):
```

Comment:
```suggestion
    def test_create_with_invalid_offerer(self, authenticated_client):
```
Et en général, je mettrais `test_create_with_yadayada`, à mon opinion